FinniversKit/Sources/DNA/Color/ColorProvider.swift

```diff
@@ -12,9 +12,21 @@ public protocol ColorProvider {
     var bgBottomSheet: UIColor { get }
     var bgAlert: UIColor { get }
     var bgInfo: UIColor { get }
+    var bgInformationAlert: UIColor { get }
+    var bgInformationCritical: UIColor { get }
+    var bgInformationInfo: UIColor { get }
+    var bgInformationSuccess: UIColor { get }
     var bgInfoHeader: UIColor { get }
     var bgSuccess: UIColor { get }
     var bgCritical: UIColor { get }
+    var borderAlert: UIColor { get }
+    var borderCritical: UIColor { get }
+    var borderInfo: UIColor { get }
+    var borderSuccess: UIColor { get }
+    var sidebarAlert: UIColor { get }
```

Comment:
What is sidebar referring to? There are quite a few new semantic colors here, we seldom add new ones. But I guess the colors you are using in this PR are not really used anywhere else yet - so they're not available in other semantic colors?

Reply:
Yeah it slowly span out of control for me as I worked my way through te design. I've added this picture that hopefully says more than words 😄 

<img width="1047" alt="Screenshot 2024-01-09 at 09 55 23" src="https://github.com/finn-no/FinniversKit/assets/117636/ed42b445-8052-42aa-8a12-fc9b5cbd5ccd">

So you have 3 required colors for the component...and you have 4 variants of the component... :sweat_smile:

I'm happy to change the name but I'm struggling to come up with something intelligent. 

I asked ChatGPT who came up with these suggestions

```
InfoBar: Since the dark blue area includes an informational icon (the "i"), "InfoBar" might be a descriptive and succinct term.
Highlight Strip: This term could work because the area stands out and serves to highlight or draw attention to the importance of the information.
Accent Panel: The term "panel" is often used in design to refer to distinct sections of a user interface, and "accent" suggests that it has a visual emphasis different from the main content area.
Status Bar: If the dark blue area is intended to indicate some kind of status or importance, "Status Bar" could be an appropriate term.
Lead-in Stripe: This term could be used if the purpose of the area is to lead the eye to the important message or if it acts as an introductory element.
Tagline: If the element is meant to tag the content with some sort of categorization or importance, "Tagline" could be suitable.
Alert Rail: If the element serves to alert the user to important information, combining "alert" with "rail" (a design term often used for side elements) might fit.
Notification Edge: This term implies that the area serves as a notification or alert to the user about something important.
Preface Sidebar: A preface generally introduces something, and this term could be used if the dark blue area serves to introduce or preface the content to follow.
Priority Bar: If the content within this dark blue area is of higher priority or importance, this name would convey that significance.
```

Accent could be a candidate 🤔

Reply:
How are these colors referenced in the design system? Maybe we can get names from there?

Reply:
Yeah, I think we can follow the the design system. Semantic colors should be reusable in other contexts, the sidebar naming here is probably only relevant for this exact view. I checked the case with yellow-600, that one seems to be called `color-background-warning` in the design system, so we could reuse something like `bgWarning`.

Reply:
Also, any colors unique to a view do not belong in the color provider. Just use a private extension along with the view, but it should be very rare that a semantic color you can use doesn't exist.

Reply:
After getting introduced to the "NMP Design Tokens taxonomy v2.0" document I made some changes so that we now have colors named:

- border<informationType>Subtle: for the border around the edges of the box
- border<InformationType>: for the left sidebar.

where informationType can be:

- critical
- information
- success
- warning

It seems these values must support light and dark mode and must be different in default and Tori configurations. That speaks to the colors being resent in the Color provider right?

Reply:
Sheeeesh! I forgot about the background colors. They got an update as well and now there is support for light and dark mode

<img width="751" alt="Screenshot 2024-01-09 at 13 25 40" src="https://github.com/finn-no/FinniversKit/assets/117636/5996cd19-ab2d-4a78-b31f-4fb7197398cd">

Reply:
ooo, nice 😍

Reply:
> It seems these values must support light and dark mode and must be different in default and Tori configurations. That speaks to the colors being resent in the Color provider right?

👍